### PR TITLE
Task OSIDB-3002: Disable private comment creation

### PR DESCRIPTION
**Pull request by C-Valen**
# [OSIDB-3002] [Disable private comments creation until it is supported by OSIDB]

## Checklist:

- [x] Linting passed
- [x] Type checks passed
- [x] Tests suite passed
- [x] Commits consolidated
- [x] Changelog updated
- [-] Test cases not added/updated
- [x] Jira ticket updated

## Summary:

Temporary disables private comment creation until it is supported from OSIDB side.

## Changes:

- Disable comment creation button on private comments tab
- Add a tooltip to inform users that the action is temporary disabled disabled
- Hide comment creation input text area on private comments tab

## Considerations:

- Didn't add reasons on the changelog as we are providing the ticket where all the info can be found

### CODE REVIEW

File: src/views/FlawEditView.vue [hunk 1]
```diff
@@ -41,9 +41,9 @@ function refreshFlaw() {
   <main>
     <FlawForm
       v-if="flaw"
+      :key="`${flaw.uuid}-${flaw.updated_dt}`"
```
Comment (MrMarble) [inline src/views/FlawEditView.vue:44]: Why this change?
It's only the order of the props?

**Review by d10n — APPROVED** [review]

File: src/views/FlawEditView.vue (hunk 1 above)
Comment (C-Valen) [reply src/views/FlawEditView.vue:44]: Yes, is only the attribute order as it appeared to me on linting warns.

**Review by JakubFrejlach — APPROVED** [review]
